app/src/main/java/com/kickstarter/libs/KoalaEvent.java

```diff
@@ -30,6 +32,8 @@ private KoalaEvent() {}
   public static final String SENT_MESSAGE = "Sent Message";
   public static final String SHOWED_SHARE_SHEET = "Showed Share Sheet";
   public static final String STARRED_PROJECT = "Starred Project";
+  public static final String TWO_FACTOR_AUTH_CONFIRM_VIEW = "Two-factor Authentication Confirm View";
+  public static final String TWO_FACTOR_AUTH_RESEND_CODE = "Two-factor Authentication Resend Code";
```

Comment:
sweet!! forgot to mention that you can now also use these static var events in Koala.java to for the actual tracking event too :neckbeard: